### ISSUE TITLE
mapState-to-props-no-store false positives on destructured state

### ISSUE THREAD

**Issue by calinoracation** [DianaSuvorova/eslint-plugin-react-redux#11]
We have quite a bit of code that destructures state as the first argument of `mapStateToProps` that then will re-export a subset of the state, I wouldn't expect the following to throw:

```jsx
const mapStateToProps = ({ header }) => ({
  isLoggedIn: header.user && header.user.isLoggedIn,
  isAdmin: header.admin && header.admin.isAdmin,
});
```

**Comment by DianaSuvorova** [comment]
Thanks for the code. Yes, this is reasonable. Will take a look.
